process/templates/SIG_NAME_incubating_stage.md

```diff
@@ -1,5 +1,32 @@
 ## Special Interest Group (SIG) incubation
 
 The governing body must agree that the SIG has made substantial progress on a deliverable.
-
    * Link to relevant documentation.
+
+### SIG has met all Sandbox requirement
+  * "link to sandbox PR"
```

Comment:
Same comment as before: I think there is no need to ask people to dig this up.

Reply:
added suggested language, thx